styles.css

```diff
@@ -0,0 +1,119 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  margin-top: 10px;
+}
+
+body {
+  background-image: url('./Images/header-shapes\ mobile.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 100px;
+  margin: 10px 30px;
+}
+
+.logo {
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  order: 0;
+  flex-grow: 0;
+  text-decoration: none;
+}
+
+.navbar {
+  display: block;
+  width: 18px;
+  height: 3px;
+  margin: 3px;
+  background-color: #6070ff;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 606px;
+  align-items: center;
+  padding-top: 50px;
+  padding-right: 20px;
+}
+
+.title {
+  height: 150px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: stretch;
+  color: #172b4d;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 30px;
+  margin-right: 20px;
+}
+
+.intro {
+  height: 150px;
+  font-family:
+    'Poppins',
+    sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex-wrap: wrap;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 40px;
+  padding-left: 40px;
+  padding-right: 30px;
+}
+
+.connect {
+  height: 24px;
+  font-family:
+    'Poppins',
+    sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  order: 2;
+  flex-grow: 0;
+  justify-content: flex-start;
+  text-decoration: none;
+  margin-top: 10px;
+  margin-bottom: 20px;
+}
+
+.social-media-icons {
+  height: 20px;
+  order: 99;
+  display: flex;
+  justify-content: flex-start;
+  gap: 1rem;
+  margin-right: 200px;
+}
```

Comment:
- [ ] Kindly make sure to center the social media icon in the container in order to meet the provided Figma design. :wink: 

   - ### YOUR DESIGN
     ![image](https://user-images.githubusercontent.com/87197412/205848468-43c64b17-0858-451a-ad4d-95ee9e26659d.png)


    - ### FIGMA DESIGN 
         ![image](https://user-images.githubusercontent.com/87197412/205848596-d8790600-1a16-45ba-8f4b-f1deefa63b13.png)